content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -120,13 +120,13 @@ StorageClass has the field `allowVolumeExpansion` set to true.
 
 {{< table caption = "Table of Volume types and the version of Kubernetes they require"  >}}
 
-| Volume type          | Required Kubernetes version |
-| :------------------- | :-------------------------- |
-| rbd                  | 1.11                        |
-| Azure File           | 1.11                        |
-| Portworx             | 1.11                        |
-| FlexVolume           | 1.13                        |
-| CSI                  | 1.14 (alpha), 1.16 (beta)   |
+| Volume type          | Required Kubernetes version              |
+| :------------------- | :--------------------------              |
+| rbd                  | 1.11                                     |
+| Azure File           | 1.11                                     |
+| Portworx             | 1.11                                     |
+| FlexVolume           | 1.13                                     |
+| CSI                  | 1.14 (alpha), 1.16 (beta), 1.24 (stable) |
```

Comment:
```suggestion
| CSI                  | 1.24 |
```
This is simpler, and we don't support any older version of Kubernetes.

Reply:
(WDYT?)

Reply:
True, good point. I was tempted to do that, but wasn't sure if we needed to keep older legacy information like that around. I will simplify it to just state the 1.24 release.

Side note, it may be a good "help-wanted/good-first-issue" task for someone looking to get involved to start culling some of those obsolete version statements from the docs.